cmd/boulder-ca/main.go

```diff
@@ -61,15 +62,26 @@ type Config struct {
 		}
 
 		// How long issued certificates are valid for.
-		// Deprecated: Use Issuace.CertProfiles.MaxValidityPeriod instead.
+		// Deprecated: Use Issuance.CertProfiles.MaxValidityPeriod instead.
 		Expiry config.Duration
 
 		// How far back certificates should be backdated.
-		// Deprecated: Use Issuace.CertProfiles.MaxValidityBackdate instead.
+		// Deprecated: Use Issuance.CertProfiles.MaxValidityBackdate instead.
 		Backdate config.Duration
 
 		// What digits we should prepend to serials after randomly generating them.
-		SerialPrefix int `validate:"required,min=1,max=127"`
+		// Deprecated: Use SerialPrefixHex instead.
+		SerialPrefix int `validate:"required_without=SerialPrefixHex,omitempty,min=1,max=127"`
+
+		// What hex string we should prepend to serials after randomly
+		// generating them. The minimum value is "01" to ensure that at least
+		// one bit in the prefix byte is set. The maximum value is "7f" to
+		// ensure that the first bit in the prefix byte is not set. The validate
+		// library cannot enforce mix/max values on strings, so that is done in
+		// NewCertificateAuthorityImpl.
```

Comment:
Field comments should start with the field name, followed by a complete sentence where the field name is the subject. While the field comments above this one don't follow this style, the one that we're adding should.

We don't uniformly rely on the validator, so I'm not sure it makes sense to call out where the field is eventually validated. I left that bit out of the suggestion below but you can feel free to put it back.

```suggestion
		// SerialPrefixHex is the hex string to prepend to serials after randomly
		// generating them. The minimum value is "01" to ensure that at least one
		// bit in the prefix byte is set. The maximum value is "7f" to ensure that
		// the first bit in the prefix byte is not set. 
		//
		// TODO(#7213): Replace `required_without` with `required` when
		// SerialPrefix is removed.
```

Reply:
Fixing, thanks! I'd like to leave the validation caveat in here because, when we tried to use min/max, the results were surprising and mildly exciting (there was some digging through a stack trace) and so this comment would have saved past-me a little mild consternation.